dbschema/committees_v2.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE committees_v2 (
```

Comment:
I appreciate that we're proactively defining these tables now instead of relying on csvsql to automagically do it for us.

Reply:
Me, too.  It does forces us to be more aware of the data.